workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/model/JiraSupportIssue.java

```diff
@@ -33,15 +35,37 @@ public JiraSupportIssue(JSONObject jsonObject, String ticketURL) {
 
 		_labels = labels.toArray(new String[0]);
 
-		JSONObject statusJSONObject = fieldsJSONObject.getJSONObject("status");
+		_organizationObjectId = StringPool.BLANK;
```

Comment:
organization isn't a jira object (aka asset), so it should just be organizationId